examples/service/ocean_vng/providers/aws/create/main.go

```diff
@@ -0,0 +1,69 @@
+package main
```

Comment:
please change the folder structure.
service/ocean/providers/aws/vng/create

Reply:
Changed example file directory as requested.